solanches/controller.py

```diff
@@ -176,3 +176,23 @@ def remove_produto_destaques(comercio_nome, produto_id):
     cardapio = get_cardapio(comercio_nome)
     return cardapio
 
+
+def adiciona_categoria(comercio_nome, categoria):
+    assert categoria, 'Erro: categoria vazia!'
```

Comment:
Como não é um parâmetro que tem valor default, então ele nunca entraria nesse assert. Talvez seja melhor verificar o tipo do parâmetro
```suggestion
    assert type(categoria) is str, 'Erro: valor de categoria inválido!'
```